doc/source/learning/overview.rst

```diff
@@ -0,0 +1,92 @@
+Overview
+=====================
+
+CaaSP v3 Install
+-----------------
+
+At this moment we need CaaSP v3 for our deployments. This is our SUSE supported Kubernetes
+deployment. The minimum number of nodes needed for the install is 4: 1 admin node, 1 kubernetes
+master node and 2 kubernetes worker nodes.
+
+To install CaaSP you can find a `deployment guide`_ and a `quick install`_ guide on SUSE online
+documentation.
+
+.. _deployment guide: https://www.suse.com/documentation/suse-caasp-3/singlehtml/book_caasp_deployment/book_caasp_deployment.html
+.. _quick install: https://www.suse.com/documentation/suse-caasp-3/singlehtml/book_caasp_installquick/book_caasp_installquick.html
+
+For development environments we are deploying CaaSP on top of OpenStack VMs. The best way to deploy
+several VMs together is through Heat teamplates. You can find a `specific section`_ on how to do that in our online documentation.
+
+.. _specific section: https://www.suse.com/documentation/suse-openstack-cloud-8/book_deployment/data/install_heat_templates.html
+
+Heat templates for CaaSP v3 need an `image`_ specific for CaaSP based on `MicroOS`_. Heat template
+will automatically download this image but you may need to download it from an
+`alternative location`_ if this one doesn't work.
+
+.. _MicroOS: https://en.opensuse.org/Kubic:MicroOS
+.. _image: http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2
+.. _alternative location: https://www.suse.com/documentation/suse-openstack-cloud-8/book_deployment/data/sect1_9_chapter_book_deployment.html
+
+
+The script run.sh
```

Comment:
I don't think this is already documented in the deploy guide.

Reply:
This wasn't in the deploy guide, but probably it is not mandatory to explain it. Nevertheless, the CaaSP v3 whole section has been removed because: 1) it was somehow covered in the deploy guide 2) will be moved Soon(TM) to CaaSP v4.